tests/observability/metrics/test_workqueue_metrics.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+
+from utilities.constants import KUBEVIRT_WORKQUEUE_QUEUE_DURATION_SECONDS, KUBEVIRT_WORKQUEUE_WORK_DURATION_SECONDS
+
+CNV_WORKQUEUE_METRICS = [
+    "kubevirt_workqueue_adds_total",
+    "kubevirt_workqueue_depth",
+    "kubevirt_workqueue_longest_running_processor_seconds",
+    f"{KUBEVIRT_WORKQUEUE_QUEUE_DURATION_SECONDS}_bucket",
+    f"{KUBEVIRT_WORKQUEUE_QUEUE_DURATION_SECONDS}_sum",
```

Comment:
personally, I think it loses readability when you use constants here..

KUBEVIRT_WORKQUEUE_QUEUE_DURATION_SECONDS and KUBEVIRT_WORKQUEUE_WORK_DURATION_SECONDS not used directly, only as a substring in other metrics